monitoring/backups_collector.go

```diff
@@ -0,0 +1,98 @@
+package monitoring
+
+import (
+	"strconv"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/longhorn/longhorn-manager/datastore"
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
+)
+
+type BackupsCollector struct {
+	*baseCollector
+
+	sizeMetric  metricInfo
+	stateMetric metricInfo
+}
+
+func NewBackupsCollector(
+	logger logrus.FieldLogger,
+	nodeID string,
+	ds *datastore.DataStore) *BackupsCollector {
+
+	vc := &BackupsCollector{
+		baseCollector: newBaseCollector(subsystemBackups, logger, nodeID, ds),
+	}
+
+	vc.sizeMetric = metricInfo{
+		Desc: prometheus.NewDesc(
+			prometheus.BuildFQName(longhornName, subsystemBackups, "actual_size_bytes"),
+			"Actual size of this backup",
+			[]string{volumeLabel, backupsLabel},
+			nil,
+		),
+		Type: prometheus.GaugeValue,
+	}
+
+	vc.stateMetric = metricInfo{
+		Desc: prometheus.NewDesc(
+			prometheus.BuildFQName(longhornName, subsystemBackups, "state"),
+			"State of this backup",
+			[]string{volumeLabel, backupsLabel},
+			nil,
+		),
+		Type: prometheus.GaugeValue,
+	}
+
+	return vc
+}
+
+func (vc *BackupsCollector) Describe(ch chan<- *prometheus.Desc) {
+	ch <- vc.sizeMetric.Desc
+	ch <- vc.stateMetric.Desc
+}
+
+func (vc *BackupsCollector) Collect(ch chan<- prometheus.Metric) {
+	defer func() {
+		if err := recover(); err != nil {
+			vc.logger.WithField("error", err).Warn("panic during collecting metrics")
+		}
+	}()
+
+	backupsLists, err := vc.ds.ListBackupsRO()
+	if err != nil {
+		vc.logger.WithError(err).Warn("error during scrape ")
```

Comment:
nit
```suggestion
                vc.logger.WithError(err).Warn("error during scrape")
```

Reply:
Fix in b2145965748e3ec8da25eec1af7c15c48aab3d68